crates/fiber-lib/src/cch/order.rs

```diff
@@ -71,12 +72,10 @@ pub struct SendBTCOrder {
     pub wrapped_btc_type_script: ckb_jsonrpc_types::Script,
 
     pub btc_pay_req: String,
-    pub ckb_pay_req: String,
+    pub fiber_payee_pubkey: Pubkey,
+    pub fiber_pay_req: String,
```

Comment:
from the code, I found it's a fiber invoice, maybe `fiber_pay_invoice` is a better name?

Reply:
it's better to use `Option<CkbInvoice>` here so we don't need to convert between string format.
or seems it's better if we generate `CkbInvoice` in SendBTCOrder constructor(Perhaps there will be some redundant code for processing parameters), so we can use `CkbInvoice` here.